addons/html_builder/static/src/core/utils.js

```diff
@@ -415,26 +416,11 @@ function useReloadAction(getAllActions) {
     return { reload };
 }
 
-export function useHasPreview(getAllActions) {
+export function useHasPreview() {
     const comp = useComponent();
-    const reload = useReloadAction(getAllActions).reload;
-    const getAction = comp.env.editor.shared.builderActions.getAction;
-
-    let hasPreview = true;
-    for (const descr of getAllActions()) {
-        if (descr.actionId) {
-            const action = getAction(descr.actionId);
-            if (action.preview === false) {
-                hasPreview = false;
-            }
-        }
-    }
-
     return (
-        hasPreview &&
-        !reload &&
-        (comp.props.preview === true ||
-            (comp.props.preview === undefined && comp.env.weContext.preview !== false))
+        comp.props.preview === true ||
```

Comment:
why you don t need to use action.preview ?

Reply:
Maybe keep this change in anoter PR. I m not sure we want it. If we really need it in the futur we will merge it.